client/__tests__/util/stateManager/sampleResponses.ts

```diff
@@ -75,6 +75,15 @@ const aSchemaResponse: { schema: RawSchema } = {
       obs: [{ name: "umap", type: "float32", dims: ["umap_0", "umap_1"] }],
       var: [],
     },
+    uns: {
```

Comment:
can we add a comment here to tell the readers that the response here should always match the `if spatial is None` case in `def spatial_metadata_get(spatial)` in case the response shape changes in the future?

Also add a comment in `def spatial_metadata_get(spatial)` to remind us to update `sampleResponses.ts` when the response shape changes

Thank you!